### ISSUE TITLE
Error: Could not find or load main class com.tplink.omada.start.OmadaLinuxMain

### ISSUE THREAD

**Issue by al3x1337** [MattFryer/Hassio-Addons#4]
```
INFO: Time zone set to 'America/Toronto'
INFO: Starting Omada Controller as user omada
tail: cannot open '/opt/tplink/EAPController/logs/server.log' for reading: No such file or directory
Error: Could not find or load main class com.tplink.omada.start.OmadaLinuxMain
```

I get this error when disabling ssl and starting the container. I've checked with Portainer to make sure there are no duplicate containers and have deleted all the Omada images before reinstalling this addon. Error still persists.

**Comment by Antexa** [comment]
Same issue here.

```
INFO: Time zone set to 'Europe/Paris'
INFO: Starting Omada Controller as user omada
tail: cannot open '/opt/tplink/EAPController/logs/server.log' for reading: No such file or directory
Error: Could not find or load main class com.tplink.omada.start.OmadaLinuxMain
```

**Comment by al3x1337** [comment]
I have tried on a fresh Hassio installation and the same error occurs.

**Comment by MattFryer** [comment]
I'll take a look at replicating this. In the interim you should be able to run it with SSL turned on without providing a certificate. Omada controller will use a self signed certificate. You'll just need to bypass the browser warning when accessing.

**Comment by Antexa** [comment]
Hi @MattFryer, 

I rollback to 4.4 image and it works as expected. It's on mbentley side. 

`ARG BASE=mbentley/omada-controller:4.4
`

**Comment by MattFryer** [comment]
OK great to know. I'll fix the base image to 4.4 for now.
